WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1592,60 +1531,34 @@ extension ReaderStreamViewController : WPTableViewHandlerDelegate {
             displayNoResultsView()
         } else {
             hideResultsStatus()
+            tableView.flashScrollIndicators()
         }
     }
 
 
     // MARK: - TableView Related
 
     public func tableView(tableView: UITableView, estimatedHeightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        // When using UITableViewAutomaticDimension for auto-sizing cells, UITableView
+        // likes to reload rows in a strange way.
+        // It uses the estimated height as a starting value for reloading animations.
+        // So this estimated value needs to be as accurate as possible to avoid any "jumping" in
+        // the cell heights during reload animations.
+        // Note: There may (and should) be a way to get around this, but there is currently no obvious solution.
+        // Brent C. August 8/2016
+        if let height = estimatedHeightsCache.objectForKey(indexPath) as? CGFloat {
+            // Return the previously known height as it was cached via willDisplayCell.
+            return height
+        }
         return estimatedRowHeight
     }
 
 
     public func tableView(aTableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
```

Comment:
Looks funky, I know, but we need the actual method since we're using `WPTableViewHandler` that asks for it via the method. However, I'm removing setting both `.estimatedRowHeight` and `.rowHeight` as it's unnecessary.